skyflow/service_account/_token.py

```diff
@@ -56,8 +59,11 @@ def generate_bearer_token_from_creds(credentials: str) -> ResponseToken:
     `credentials` arg takes the content of the credentials file that is downloaded after Service Account creation.
```

Comment:
Gets a bearer token for a service account. \`credentials` is the contents of the credentials file that your browser downloads after creating a service account.